### PR TITLE
define HTMLOrSVGElement mixin

### PR DESCRIPTION
**Pull request by siusin**
Fix issue #1299 .

Conform with the current WPT [html idl](https://github.com/web-platform-tests/wpt/blob/master/interfaces/html.idl#L124) test.

Also to fix a bikeshed building error in the editing host session.

### CODE REVIEW

**Comment by chaals** [comment]
Build failed... can you please check?

**Comment by siusin** [comment]
We'll need to upgrade the Bikeshed tool we used in the build script, otherwise it will report an error on IDL mixin.

**Review by chaals — APPROVED** [review]
Looks good - thank you

File: sections/changes.include [hunk 1]
```diff
@@ -13,6 +13,9 @@
 <h3 id="changes-wd4">Changes since the <a href="https://www.w3.org/TR/2018/WD-html53-20180426/">HTML 5.3 Third Working Draft</a></h3>
 <dl>
   <dt><a href="https://github.com/w3c/html/pull/1479"><code>http-equiv="set-cookie"</code> has no effect</a>.</dt>
+  <dt><a href="https://github.com/w3c/html/pull/1489">Define <code>HTMLOrSVGElement</code> mixin interface</a>.</dt>
+  <dd>Fixed <a href="https://github.com/w3c/html/issues/1299">issue 1299</a>
```
Comment (chaals) [inline sections/changes.include:17]: Should this mention moving the nonce handling? If so, maybe tweak that - merging now anyway...

**Pull request merged**